packages/opencensus-web-all/src/export-initial-load.ts

```diff
@@ -66,7 +70,9 @@ export function exportRootSpanAfterLoadEvent() {
 function exportInitialLoadSpans() {
   setTimeout(() => {
     const spanContext = getInitialLoadSpanContext();
-    if (!isSampled(spanContext)) return; // Don't export if not sampled.
+    // Set the span context to allow passing it around OpenCensus Web.
+    setInitialLoadSpanContext(spanContext);
```

Comment:
What would you think about moving this `setInitialLoadContext` over to the `getInitialLoadSpanContext` function? So that it just sets it the first time you get it?